unit/Makefile

```diff
@@ -70,6 +70,7 @@ SRC += analyses/ai/ai.cpp \
        util/small_map.cpp \
        util/small_shared_two_way_ptr.cpp \
 	util/std_expr.cpp \
+	   util/string2int.cpp \
```

Comment:
The indents here look pretty weird (likely tab/space mix) - both the newly added line as well as the one above.